From: lxl6125 <lxl6125@yeah.net>
Date: Fri, 24 Sep 2021 01:08:53 +0800
Subject: Fix dma-buf error
 Fix errors: aarch64-openwrt-linux-musl-ld: drivers/built-in.a: member drivers/dma-buf/heaps in archive is not an object

Signed-off-by: lxl6125 <lxl6125@yeah.net>
---
 drivers/dma-buf/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletions(-)

--- a/drivers/dma-buf/Makefile
+++ b/drivers/dma-buf/Makefile
@@ -4,7 +4,7 @@
 dma-buf-objs-y := dma-buf.o dma-fence.o dma-fence-array.o dma-fence-chain.o \
 	 dma-resv.o seqno-fence.o
 dma-buf-objs-$(CONFIG_DMABUF_HEAPS)	+= dma-heap.o
-dma-buf-objs-$(CONFIG_DMABUF_HEAPS)	+= heaps/
+obj-$(CONFIG_DMABUF_HEAPS)		    += heaps/
 dma-buf-objs-$(CONFIG_SYNC_FILE)		+= sync_file.o
 dma-buf-objs-$(CONFIG_SW_SYNC)		+= sw_sync.o sync_debug.o
 dma-buf-objs-$(CONFIG_UDMABUF)		+= udmabuf.o
